src/NuGetForUnity/Editor/PackageContentManager.cs

```diff
@@ -173,6 +98,65 @@ internal static void CleanInstallationDirectory(INugetPackageIdentifier package)
             }
         }
 
+        internal static bool ShouldSkipUnpackingOnPath(string path, string packageId)
+        {
+            // skip a remnant .meta file that may exist from packages created by Unity
+            if (path.EndsWith($"{packageId}.nuspec.meta", StringComparison.Ordinal)) return true;
+
+            // skip directories & files that NuGet normally deletes
+            if (path.StartsWith("_rels", StringComparison.Ordinal) || path.Contains("/_rels/")) return true;
+            if (path.StartsWith("package", StringComparison.Ordinal) || path.Contains("/package/")) return true;
+            if (path.EndsWith($"{packageId}.nuspec", StringComparison.Ordinal)) return true;
+            if (path.EndsWith("[Content_Types].xml", StringComparison.Ordinal)) return true;
+
+            // Unity has no use for the build directory
+            if (path.StartsWith("build", StringComparison.Ordinal) || path.Contains("/build/")) return true;
```

Comment:
```suggestion
            if (path.StartsWith("build/", StringComparison.Ordinal) || path.Contains("/build/")) return true;
```
The `path` is always a file path so to match a directory name we need the `/`